### ISSUE TITLE
/place endpoint GID update

### ISSUE THREAD

**Issue by dianashk** [pelias/api#414]
### ID generation & the /place endpoint

_tl;dr we should review and discuss this, it's still not quite right._

the id syntax has changed from `osm:venue:50133` to `openstreetmap:venue:node:2688438317`, the tests need to be updated.

**remark:** the id syntax is `{source_name}{pelias_layer}{source_layer}{source_id}`, would it be better to group the 'source' attributes at the start and have the 'pelias' attributes following? This would allows us the flexibility to add more 'pelias' fields at a later date, it would also be easier to read/substring when quickly scanning for source info.

**remark:** some ids seem to be composed of 3 component instead of 4, such as "gid": "whosonfirst:locality:101932657".

**note:** this is a breaking change and should be communicated to current users, also to the mapzen mobile teams who are currently working on supporting this API.

http://pelias.github.io/compare/#/v1/place%3Fids=osm:venue:5013364
http://pelias.github.io/compare/#/v1/place%3Fids=openstreetmap:venue:node:2688438317

**Comment by orangejulius** [comment]
I happened upon this issue which was un-resolved, but I think in the time since then we have realized it is okay. The official Pelias GID format is `source:layer:id`. However, OSM effectively needs a prefix to their integer IDs to identify which type of record it is. Perhaps choosing `:` as that separator is unclear (it could be changed to `/` to correspond to the format of URLs for OSM records), but overall the format seems good.

If we ever need to revise this decison, feel free to comment or re-open.
